include/MainWindow.h

```diff
@@ -81,9 +81,20 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	const static int m_autoSaveLongSeconds = 120;
+	const static int m_autoSaveShortSeconds = 10;
+	const static int m_autoSaveLongTime = m_autoSaveLongSeconds * 1000;
+	const static int m_autoSaveShortTime = m_autoSaveShortSeconds * 1000;
+
+	void autoSaveTimerReset( int msec = m_autoSaveLongTime )
+	{
+		m_autoSaveTimer.start( msec );
+	}
+
+	int getAutoSaveTimerInterval()
 	{
-		m_autoSaveTimer.start( 1000 * 60 );  // 1 minute
+		return m_autoSaveTimer.interval() / 1000;
```

Comment:
That may actually be a remnant from the debug info I tucked in there so I could work with seconds only and spare my brain a bit. I'll go over it again.